shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-postgresql/src/main/java/org/apache/shardingsphere/proxy/frontend/postgresql/auth/PostgreSQLAuthenticationEngine.java

```diff
@@ -91,7 +91,8 @@ public AuthenticationResult auth(final ChannelHandlerContext context, final Pack
             }
             md5Salt = PostgreSQLRandomGenerator.getInstance().generateRandomBytes(4);
             context.writeAndFlush(new PostgreSQLAuthenticationMD5PasswordPacket(md5Salt));
-            return AuthenticationResult.continued(username, databaseName);
+            currentAuthResult = AuthenticationResult.continued(username, databaseName);
```

Comment:
Can remove currentAuthResult?

Reply:
Probably not, it's latest code after refactoring.